docs/RFCs/package-toml.md

```diff
@@ -0,0 +1,166 @@
+manifest file due to conflicts arising from the namespacing entropic uses.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Copyright
+
+This document is copyright by its individual contributors, and available for
+re-use under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+## Introduction
+
+Entropic cannot share a manifest format with the traditional `package.json` format
+associated with the ECMAScript ecosystem for a number of reasons:
+
+1. Entropic is intended to allow for federated package management for more than just the ECMAScript ecosystem
+1. The `package.json` name specifier does not allow for the namespacing that entropic provides
+1. There are recognized deficiencies with using JSON as human-readable metadata format
+
+The project has chosen to standardize on TOML as a format due to various short-comings in the JSON file specification, especially concerns around:
+
+1. Lack of comments
+1. Lack of multiline string support
+1. Designed for ease of generation by machine, not by hand
+
+## Filename
+
+The filename of the manifest **MUST** be `[TK].toml` and it **MUST** live in
+the root directory of the package's filesystem. The root of the package is defined
+as the closest parent directory that contains the manifest file, starting with
+the current directory.
+
+<details>
+<summary>If file name is case-sensitive</summary>
+On case-insensitive filesystems, it **MAY** be presented as `[TK].toml` but care
+should be taken to preserve case when possible.
+</details>
+
+## Format
+
+The format of this file is **REQUIRED** to be in [TOML](https://github.com/toml-lang/toml)
+version 0.5.0 or later as defined by the project.
+
+### Top-Level Data
+
+The manifest **MUST** contain the following top-level keys, in order to be published:
+
+- `"name"` - the canonical name of the package, as a string, including its namespace. e.g. `"toddself@static-pkg.dev/packge2toml"`
+- `"version"` - a [SEMVER](https://semver.org/) designation conforming to version 2.0.0 of the semantic versioning specification. Due to how TOML parses numbers, this must be represented as a string. e.g. `"1.0.4"`
+
+These keys are **NOT REQUIRED** be present when interacting with the CLI locally (e.g. installing depedencies).
+
+The manifest **MAY** contain the following fields:
+
+- `"entry"` - the filename and package-relative path, as a string, to the main entry point of the package. The root for the relative-path is the directory which contains the `[TK].toml` file. e.g `"./src/index.js"`.
+- `"type"` - if set to `"module"` this will output a [`package.json` with this included](https://nodejs.org/api/esm.html#esm_code_package_json_code_code_type_code_field), and will signal to the node interpreter to run this code an ES6 Module.
```

Comment:
the type doesn't actually signal to the loader what type to load, it signifies how to interpret file extensions

You can have `pkg.type` unset, and it would default to `commonjs`. Based on this current behavior having no `type` field and a `main` with a `.mjs` extension would still export an esm module.

Similarly a `type` of `module` with a `main` with the `.cjs` extension will still export a cjs module

Reply:
clarified!